test/functional/wiki_controller_test.rb

```diff
@@ -173,7 +178,10 @@ def teardown
     newtitle = 'New Title'
 
     post :update, params: { id: wiki.nid, uid: users(:bob).id, title: newtitle, body: 'Editing about Page' }
+    # latest_activity_nid should be updated in all tags related to the wiki
```

Comment:
This all looks great. Would there be any extra benefit to some tests that assert both before and after we take an action that bumps up a topic? We could then test against the `@tag_subscriptions` variable in the home controller action?

What do you think? Thanks Ruth this is exciting!

Reply:
Hi Jeff, thanks!

That's a test I thought about but wasn't sure how to approach it, I'll definitely try using the `@tag_subscriptions`. I'll let you know once it is complete.

Reply:
We might also want to add an assertion before and after the count changes, as that'll be one way to protect the code while removing the redundant update you've taken out!

Reply:
I think you can directly access `@tag_susbcriptions` with:

```ruby
assert_equal SOME_KNOWN_VALUE, assigns[:tag_susbcriptions]
assert_equal SOME_KNOWN_VALUE_2, assigns[:tag_susbcriptions].length
assert_equal SOME_KNOWN_VALUE_3, assigns[:tag_susbcriptions].first.some_attribute
```